kalite/main/content_rating_models.py

```diff
@@ -5,14 +6,18 @@
 from securesync.models import DeferredCountSyncedModel
 
 class ContentRating(DeferredCountSyncedModel):
+
+    minversion = "0.15.0"
+
     class Meta:
         unique_together = ("content_kind", "content_id", "user")
 
     # Maintain info on content type -- could be video, exercise, etc, but we should be able to uniquely id it
     content_kind = models.CharField(max_length=100, db_index=True, blank=False)
     content_id = models.CharField(max_length=100, db_index=True, blank=False)
+    content_source = models.CharField(max_length=100, db_index=True, default=settings.CHANNEL)

```

Comment:
Yes, so does #4149. I didn't commit a migration file in anticipation of the fields changing. My thought is that we can wait for any new models to be introduced into 0.15, and then create migrations for whatever is necessary, in the interest of keeping the total number of migrations low and the history simple.